### PR TITLE
[cli] Change mentions of "Aborted" to "Canceled"

### PR DESCRIPTION
**Pull request by MatthewStanciu**
@leerob flagged that the language "Aborted" throughout the CLI should be changed to be more inclusive. This PR updates this throughout.

I didn't go as far as to update type names because I wanted to avoid potentially unintended consequences, but happy to also do that if we want.

#### Tests

- [ ] The code changed/added as part of this PR has been covered with tests
- [ ] All tests pass locally with `yarn test-unit`

#### Code Review

- [ ] This PR has a concise title and thorough description useful to a reviewer
- [ ] Issue from task tracker has a link to this PR

### CODE REVIEW

**Review by styfle — CHANGES_REQUESTED** [review]
Let's use "Canceled" with one L since that's how we spell it elsewhere.

For example https://vercel.com/blog/canceling-ongoing-deployments

File: packages/cli/src/commands/alias/rm.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export default async function rm(
 
   const removeStamp = stamp();
   if (!opts['--yes'] && !(await confirmAliasRemove(client, alias))) {
-    output.log('Aborted');
+    output.log('Canceled.');
```
Comment (cb1kenobi) [inline packages/cli/src/commands/alias/rm.ts]: I think we should be consistent whether the period is present. I would argue that if it's only a single word, then omit the period.

File: packages/cli/src/commands/build.ts [hunk 2]
```diff
@@ -171,7 +171,7 @@ export default async function main(client: Client): Promise<number> {
       );
     }
     if (!confirmed) {
-      client.output.print(`Aborted. No Project Settings retrieved.\n`);
+      client.output.print(`Cancelled. No Project Settings retrieved.\n`);
```
Comment (styfle) [inline packages/cli/src/commands/build.ts]:
```suggestion
      client.output.print(`Canceled. No Project Settings retrieved.\n`);
```

**Review by styfle — APPROVED** [review]
Thanks!

File: packages/cli/src/commands/alias/rm.ts (hunk 1 above)
Comment (MatthewStanciu) [reply packages/cli/src/commands/alias/rm.ts]: Thanks, just did this in the latest commit

**Review by cb1kenobi — APPROVED** [review]
LGTM

**Review by leerob — APPROVED** [review]
Thank you so much!

**Review by javivelasco — APPROVED** [review]

**Review by EndangeredMassa — APPROVED** [review]

**Pull request merged**